Comparison Engine/Comparison Engine/Base Classes/BarManager.cs

```diff
@@ -90,7 +90,7 @@ public Bar GetBarByLocation(string barLocation)
             {
                 return barDictionary.Values.First(x => x.barLocation == barLocation);
             }
-            catch(ArgumentNullException e)
+            catch(Exception e)
```

Comment:
Same as the previous comment regarding changing ArgumentNullException e with Exception e